daemon/internal/newrelic/app.go

```diff
@@ -146,6 +146,7 @@ type App struct {
 	HarvestTrigger      HarvestTriggerFunc
 	LastActivity        time.Time
 	Rules               MetricRules
+	PhpPackages			map[PhpPackagesKey]struct{}
```

Comment:
I think gofmt needs to be run

Reply:
This is gofmted. The rest of the struct uses spaces instead of tabs. My change uses tabs (which is [correct](https://go.dev/doc/effective_go#formatting) according to Go).

Reply:
I'm sorry but this is not true. Tabs are used for indent, not structures formatting. Moreover, `gofmt` shows that these lines (among a few others) need to be reformatted:
```
gofmt -d internal/newrelic/app.go 
diff internal/newrelic/app.go.orig internal/newrelic/app.go
--- internal/newrelic/app.go.orig
+++ internal/newrelic/app.go
@@ -146,7 +146,7 @@
        HarvestTrigger      HarvestTriggerFunc
        LastActivity        time.Time
        Rules               MetricRules
-       PhpPackages                     map[PhpPackagesKey]struct{}
+       PhpPackages         map[PhpPackagesKey]struct{}
 }
 
 func (app *App) String() string {
@@ -181,7 +181,7 @@
                info:               info,
                HarvestTrigger:     nil,
                LastActivity:       now,
-               PhpPackages:            make(map[PhpPackagesKey]struct{}),
+               PhpPackages:        make(map[PhpPackagesKey]struct{}),
        }
 }
```

Reply:
24fbebf465ba0ffd6769482348db43b12d01d1b9